exporters/ostream/test/ostream_span_test.cc

```diff
@@ -145,7 +145,7 @@ TEST(OStreamSpanExporter, PrintChangedSpanCout)
       "  description   : Test Description\n"
       "  span kind     : Client\n"
       "  status        : Ok\n"
-      "  attributes    : attr1: string\n"
+      "  attributes    : \n\tattr1: string\n"
```

Comment:
Keep separate lines separate?
```suggestion
      "  attributes    : \n\tattr1: string\n"
      "\tattr1: string\n"
```

Reply:
Thanks, fixed now.